megamek/src/megamek/common/Entity.java

```diff
@@ -9350,10 +9356,19 @@ public String statusToString(int loc) {
             return "No location given.";
         }
 
-        return getLocationName(loc) + " (" + getLocationAbbr(loc)
+        String str = getLocationName(loc) + " (" + getLocationAbbr(loc)
                + "): Armor: " + getArmorString(loc) + "/" + getOArmor(loc)
                + " Structure: " + getInternalString(loc) + "/"
-               + getOInternal(loc);
+               + getOInternal(loc) + "\n ";
+        for(CriticalSlot cs : crits[loc]) {
+            if(cs != null) {
+                Mounted mount = cs.getMount();
```

Comment:
There may also be a second mount if `cs.getMount2()` is non-null. This may be where bombs for a bomb bay are mounted, or a super-heavy double slot.

Reply:
noted, do you have a better way to denote criticals for accessibility users? I am not happy with the current implementation, it's just better than nothign.

Reply:
I noodled on this for a while, but came to the conclusion that I do not have a better answer.